images.CI/lint-shebang.ps1

```diff
@@ -0,0 +1,38 @@
+param(
+    [Parameter(Mandatory)] [string] $path,
+    [Parameter(Mandatory)] [string] $pattern
+)
+
+$ErrorActionPreference = "Stop"
+
+function Validate-Scripts {
+    Param (
+        [Parameter(Mandatory=$true)]
+        [string[]]$Path,
+        [Parameter(Mandatory=$true)]
+        [string]$Pattern
+    )
+     $ScriptWithoutShebangLine = @()
+     Get-ChildItem $path | ForEach-Object {
+       if (Get-Content -Path $($_.FullName) | Select-Object -First 1 | Select-String -Pattern $Pattern -Quiet) {
```

Comment:
Why we use `Select-String`? This function check that string satisfies regex. In our case, string should be exactly like we want.
```
$shebangLine = Get-Content -Path $($_.FullName) | Select-Object -First 1
if ($shebangLine -eq $Pattern) {
```

Reply:
Thank you, I have fixed this.